app/uk/gov/hmrc/plasticpackagingtax/registration/config/AppConfig.scala

```diff
@@ -132,4 +132,9 @@ class AppConfig @Inject() (config: Configuration, val servicesConfig: ServicesCo
       .getOptional[Boolean](s"features.liabilityPreLaunch")
       .getOrElse(false)
 
+  lazy val minimumWeight: Long =
+    config
+      .getOptional[Long](s"minimum.weight")
+      .getOrElse(10000)
```

Comment:
We might need to check for places where we've hardcoded this value. I know I have in some recent changes.